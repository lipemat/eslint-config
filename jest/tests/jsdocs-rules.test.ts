import jestRunnerEslint from '../../jest/helpers/jest-runner-eslint';

describe( 'JSDoc rules are enabled', () => {
	test( 'jsdoc/check-access', async () => {
		expect( await jestRunnerEslint( 'failing/jsdoc/check-access' ) ).toMatchSnapshot();
	} );
} );
