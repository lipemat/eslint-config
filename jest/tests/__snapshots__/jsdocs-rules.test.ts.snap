// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSDoc rules are enabled jsdoc/check-access 1`] = `
"FAIL jest/fixtures/failing/jsdoc/check-access/incorrect-access.ts
  × jsdoc/check-access (mocked ms)
  × jsdoc/check-access (mocked ms)
  × jsdoc/check-access (mocked ms)


E:\\SVN\\the-boilerplate\\packages\\eslint-config\\jest\\fixtures\\failing\\jsdoc\\check-access\\incorrect-access.ts
   2:1  error  Missing valid JSDoc @access level                               jsdoc/check-access
   8:1  error  At most one access-control tag may be present on a jsdoc block  jsdoc/check-access
  16:1  error  At most one access-control tag may be present on a jsdoc block  jsdoc/check-access

✖ 3 problems (3 errors, 0 warnings)

Test Suites: 1 failed, 1 total
Tests:       3 failed, 3 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;

exports[`JSDoc rules are enabled jsdoc/require-param 1`] = `
"PASS jest/fixtures/passing/jsdoc/require-param/rule-disabled.ts
  √ ESLint (mocked ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
